src/olympia/shelves/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from olympia.shelves.models import Shelf, ShelfManagement
+
+
+class ShelfSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Shelf
+        fields = ['id', 'title', 'endpoint', 'criteria',
+                  'footer_text', 'footer_pathname']
+
+
+class HomepageSerializer(serializers.ModelSerializer):
+    shelf = ShelfSerializer(required=True)
+
```

Comment:
The next issue will be to actually include the contents of the shelves in the response.